extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -329,14 +331,17 @@ export class Bind {
       }
 
       // Remove the bindings from the evaluator.
-      if (this.workerExperimentEnabled_) {
-        dev().fine(TAG, `Asking worker to parse expressions...`);
-        return invokeWebWorker(this.win_,
-          'bind.removeBindingsWithExpressionStrings',
-          [deletedExpressions]);
-      } else {
-        this.evaluator_.removeBindingsWithExpressionStrings(deletedExpressions);
+      if (deletedExpressions.length > 0) {
+        if (this.workerExperimentEnabled_) {
+          dev().fine(TAG, `Asking worker to remove expressions...`);
+          return invokeWebWorker(
+              this.win_, 'bind.removeBindings', [deletedExpressions]);
```

Comment:
The method called on the evaluator is called `removeBindingsWithExpressionStrings`. Maybe this string should stay as it was and the string on the web worker needs to be updated?